### PR TITLE
Organization Resource getOne method

### PR DESCRIPTION
**Pull request by silvistershebin**
### Short description of what this resolves:
Added getOne method in FHIR Organization Resource

### CODE REVIEW

**Comment by sjpadgett** [comment]
@silvistershebin @yashrajbothra 
We need to make a decision concerning using prefix to ids. I originally used them for my Provider client in case we wanted to allow storing patient data on outside servers. I then refactored them out because it would save from having to parse the id whenever we used standard services or communicated with openemr database.
Having them sure makes it easier to test against public servers but we'd have to be vigilant in our interactions with resources 

thoughts?

**Comment by yashrajbothra** [comment]
I don't think so we need any prefixes. We can add [Identifier](https://www.hl7.org/fhir/datatypes.html#Identifier) in future which will be more clear and precise.

**Comment by silvistershebin** [comment]
Yeah, we can do without the prefix.

**Comment by silvistershebin** [comment]
I'll push the code with the changes

File: src/Services/FhirOrganizationService.php [hunk 1]
```diff
@@ -39,16 +39,27 @@ public function getAll()
...
         }
         return $results;
     }
-    
+
+    public function getOne($oid)
+    {
+        $facilitySQL = "SELECT id, name, phone, street, city, state, postal_code, country_code as country, email FROM facility WHERE id = ?";
+
+        $result = sqlQuery($facilitySQL, $oid);
```
Comment (sjpadgett) [inline src/Services/FhirOrganizationService.php]: we want to stay consistent and use standard service when possible. Code duplication is not needed in this resource. Take a look at how Yash did the patient service and others when he broke them out of FHIRResourceService.
I explained why in his PR...

Comment (silvistershebin) [reply src/Services/FhirOrganizationService.php:57]: @sjpadgett Sir, so should I use the existing FacilityService class? In that case, the getAll and getOne queries would return all the columns (~35) of the facility table; but we need only the id, name, contact details and address values.

Comment (sjpadgett) [reply src/Services/FhirOrganizationService.php:57]: really, that many:) No, you can leave it as you have.

**Comment by silvistershebin** [comment]
Do I need to make any other changes?

**Comment by sjpadgett** [comment]
While I was holding you up here, @yashrajbothra and I moved service and controller location on you!
PR#3298.
Depending how I merge the two PR, somebody will end up with a conflict. I think i'll merge Yash first then we can deal with your PR merge.

Or maybe git guru @bradymiller will offer suggestion.

**Comment by sjpadgett** [comment]
@silvistershebin Look what I did to ya!:) Sorry but I got excited:)
Can you fix the moved file conflict?

**Comment by sjpadgett** [comment]
Bringing in another great contribution, thanks!

**Pull request merged**